client/extensions/woocommerce/app/settings/payments/payment-method-stripe-connect-account.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Image from 'components/image';
+import resizeImageUrl from 'lib/resize-image-url';
+
+class StripeConnectAccount extends Component {
+	static propTypes = {
+		stripeConnectAccount: PropTypes.shape( {
+			displayName: PropTypes.string,
+			email: PropTypes.string,
+			firstName: PropTypes.string,
+			isActivated: PropTypes.bool,
+			lastName: PropTypes.string,
+			logo: PropTypes.string,
+		} ),
+		onDisconnect: PropTypes.func, // TODO - require most of these props in subsequent PR
+	}
+
+	renderLogo = ( stripeConnectAccount ) => {
+		const { logo } = stripeConnectAccount;
+
+		let image = null;
+
+		if ( ! isEmpty( logo ) ) {
+			image = <Image src={ resizeImageUrl( logo, { w: 40, h: 40 } ) } className="payments__stripe-connect-account-logo" />;
+		}
+
+		return (
+			<div className="payments__stripe-connect-account-logo-container">
+				{ image }
+			</div>
+		);
+	}
+
+	renderNameAndEmail = ( stripeConnectAccount ) => {
+		const { displayName, email, firstName, lastName } = stripeConnectAccount;
+		const name = ! isEmpty( displayName ) ? displayName : `${ firstName } ${ lastName }`;
+
+		return (
+			<div className="payments__stripe-connect-account-details">
+				<span className="payments__stripe-connect-account-name">
+					<span className="payments__stripe-connect-account-name">{ name }</span>
+					<span className="payments__stripe-connect-account-email">{ email }</span>
+				</span>
+			</div>
+		);
+	}
+
+	// TODO - when we are ready to connect this for-reals, this layer may not be needed
+	onDisconnect = ( event ) => {
+		event.preventDefault();
+		if ( this.props.onDisconnect ) {
+			this.props.onDisconnect();
+		}
+	}
+
+	renderStatus = () => {
+		const { stripeConnectAccount, translate } = this.props;
+		const { isActivated } = stripeConnectAccount;
+
+		let status = <span className="payments__stripe-connect-account-status account-activated">
```

Comment:
Nit: `let status = null` here and then setting the proper JSX below in an if/else format would save some JavaScript objects being created and then immediately thrown away.

Reply:
Hmmm - good point - I'll change this I think

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/17197/commits/4c6c545a3975ac1f5943dbfbe92b93c35544a937